### PR TITLE
Add Taskord to the projects list.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes gitpod-io/contribute.dev#112

For now, we have decided just to update the component to support Gitlab projects and add the project manually we will work on updating the `add-project` script later.

image

### CODE REVIEW

**Comment by nisarhassan12** [comment]
@bigint Currently I'm unable to find a way to incorporate GitLab stars badge i.e currently not found is shown
image

I found the below which can help but it requires the Gitlab PROJECT_ID would be great if you can help me with that. Thanks

https://img.shields.io/badge/dynamic/json?color=green&label=gitlab%20stars&query=%24.star_count&url=https%3A%2F%2Fgitlab.com%2Fapi%2Fv4%2Fprojects%2F[PROJECT_ID]`

**Comment by bigint** [comment]
Hi @nisarhassan12 Here is the star badge for Taskord



```
https://img.shields.io/badge/dynamic/json?color=green&label=gitlab%20stars&query=star_count&url=https://gitlab.com/api/v4/projects/20359920
```

File: src/data/projects.json [hunk 1]
```diff
@@ -622,9 +622,23 @@
...
+      "repo": "https://github.com/docToolchain/aoc-2019",
       "logo": "advent-of-code.png"
     },
+    {
+      "title": "Taskord",
+      "description": "Get things done in public with awesome community of makers.",
+      "language": "PHP",
+      "tags": [
+        {
+          "href": "https://discord.gg/9M4Q65b",
```
Comment (bigint) [inline src/data/projects.json:634]:
```suggestion
          "href": "https://taskord.com",
```

cc: @nisarhassan12

**Comment by nisarhassan12** [comment]
Thanks, @bigint for providing me with the badge.

**Comment by nisarhassan12** [comment]
This is how it looks now:

image

**Comment by bigint** [comment]
Awesome @nisarhassan12 💜

**Comment by bigint** [comment]
Thanks for merging @nisarhassan12 ❤️

**Pull request merged**